src/Forms/HTMLEditor/TinyMCEConfig.php

```diff
@@ -821,6 +855,22 @@ public static function get_tinymce_lang()
         return 'en';
     }
 
+    /**
+     * Get the URL for the language pack of the current language
+     *
+     * @return string Language
+     */
+    public static function get_tinymce_lang_url(): string
+    {
```

Comment:
The language files no longer come with tinymce - the official ones cost money and the community ones are a separate download. 
We include the community ones in the third-party dir.